src/core/plugins/icons/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @prettier
+ */
+import LargeArrowUpIcon from "./components/large-arrow-up"
```

Comment:
Would it be possible to name the icon components and export them from the plugin in more generic way? What I mean is that if we name the icons generically as for example [font awesome](https://fontawesome.com/) does it (or any other icon library), we get more reuse.

Examples:

`LockedAuthOperationIcon` -> `LockIcon`
`UnlockedAuthOperationIcon` -> `LockOpenIcon`

Reply:
As we discussed I extracted logic for changing Auth icon separately to `auth` plugin so we keep `icons` plugin simple and only responsible for changing icons.